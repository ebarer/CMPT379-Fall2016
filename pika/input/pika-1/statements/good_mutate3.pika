exec {
	var c := 3 + 4 * 5.
	const d := (3 + 4) * 5.
	print c; d, _n_.
	c := (8 + 5) * 4.
	print c; d, _n_.
	c := 0.
	print c; d, _n_.
	c := 7.
	print c; d, _n_.
}
