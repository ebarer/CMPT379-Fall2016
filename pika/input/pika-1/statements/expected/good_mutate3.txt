23 35
52 35
0 35
7 35
